pallets/gear/src/benchmarking/mod.rs

```diff
@@ -112,6 +112,7 @@ use sp_std::prelude::*;
 const MAX_PAYLOAD_LEN: u32 = 32 * 64 * 1024;
 const MAX_PAYLOAD_LEN_KB: u32 = MAX_PAYLOAD_LEN / 1024;
 const MAX_PAGES: u32 = 512;
+const MAX_SALT_SIZE_BYTES: u32 = 4 * 1024 * 1024;
```

Comment:
We have `gear_core::MAX_PAYLOAD_SIZE`

Reply:
Yes, but I use max salt size which has been presented in benchmarks before these changes